blog-posts/from-promises-to-observables/from-promises-to-observables.md

```diff
@@ -0,0 +1,390 @@
+      .pipe(
+        catchError(err => {
+          throw 'error in source observable. Message: ' + err.message;
+        })
+      );
+
+    const wordpress = 
+      this.http.get(environment.reactiveEndpoint + '/wordpress')
+      .pipe(
+        catchError(err => {
+          throw 'error in source observable. Message: ' + err.message;
+        })
+      );
+
+    const devto = 
+      this.http.get(environment.reactiveEndpoint + '/devto')
+      .pipe(
+        catchError(err => {
+          throw 'error in source observable. Message: ' + err.message;
+        })
+      );
+
+    const manualEntries = 
+      this.http.get(environment.manualEntries)
+      .pipe(
+        catchError(err => {
+          throw 'error in source observable. Message: ' + err.message;
+        })
+      );
+
+    this.posts$ =
+      merge(medium, wordpress, devto, manualEntries)
+        .pipe(
+          scan((output: Post[], response: []) => {
+            response.forEach((post: Post) => {
+              const inputDate = new Date(post.pubDate);
+              post.pubDate = inputDate.toLocaleDateString('en-us') + ' at ' + inputDate.toLocaleTimeString('en-us');
+              post.sortDate = inputDate.getTime();
+
+              if (post.sourceURL === 'https://blog.angularindepth.com/feed') {
+                post.sourceURL = 'Angular-In-Depth';
+              } else if (post.sourceURL === 'https://itnext.io/feed') {
+                post.sourceURL = 'ITNext';
+              } else if (post.sourceURL === 'https://medium.com/feed/@Andrew_Evans') {
+                post.sourceURL = 'Medium';
+              } else if (post.sourceURL === 'https://rhythmandbinary.com/feed') {
+                post.sourceURL = 'Rhythm and Binary';
+              } else if (post.sourceURL === 'https://dev.to/feed/andrewevans0102') {
+                post.sourceURL = 'DEV.TO';
+              }
+              output.push(post);
+            })
+
+            output.sort((a: any, b: any) => {
+              return b.sortDate - a.sortDate;
+            });
+
+            return output;
+        }, []),
+        catchError(err => {
+          throw 'error in source observable. Message: ' + err.message;
+        }),
+        takeUntil(this.unsubscribe)
+      );
+  }
+```
+
+Here I am taking advantage of Angular's [HttpClient](https://angular.io/guide/http) which wraps HTTP calls in an observable. 
+
+I then use the [merge](https://rxjs.dev/api/index/function/merge) operator to subscribe to all of the HttpClient calls and combine them into one output.
+
+The [scan](https://rxjs.dev/api/operators/scan) operator then takes the merged observables and appends the response to one common output.
+
+I include the [catchError](https://rxjs.dev/api/operators/catchError) operator to handle any errors in the stream, if one of the calls fails etc.
+
+I also use __pipe__ to take the output of one observable and passes it into another.  This is fairly intuitive and a common pattern with RxJS.
```

Comment:
You should remove the `fairly intuitive` here. It might not be intuitve for all your readers which  would make them feel kinda stupid